package.json

```diff
@@ -60,8 +59,6 @@
     "gzip-size": "^3.0.0",
     "inquirer": "^0.11.0",
     "jsdoc": "^3.4.0",
-    "jshint": "^2.9.1",
```

Comment:
Turns out I missed one thing – that ckeditor5-dev-lint installs jshint as its normal dependency which will work even when there are version conflicts. See also https://github.com/ckeditor/ckeditor5-dev-utils/issues/5#issuecomment-235244998.